playwright-e2e/lib/component/dsm/navigation/enums/selectStudyNav.enum.ts

```diff
@@ -16,3 +16,7 @@ export enum Study {
   RAREX = 'RareX',
   SINGULAR = 'Singular'
 }
+
+export enum Miscellaneous {
+  MAILING_LIST = 'Mailing List'
+}
```

Comment:
For sake of completeness, could you add rest of dropdown options in the enum ? It will be helpful to others.

Reply:
updated